test/test_torch.py

```diff
@@ -18225,6 +18225,40 @@ def test_nansum(self, device, dtype):
 
         self.compare_with_numpy(torch.nansum, np.nansum, x, device, dtype)
 
+    @dtypes(*(torch.testing.get_all_int_dtypes() + torch.testing.get_all_fp_dtypes(include_bfloat16=False)))
```

Comment:
I like this test but would like to reconsider using the OpInfo pattern (discussion in the conversation).

Reply:
How about keeping this test which verifies it with arguments and adding entry for `nan_to_num`  to existing `OpInfo` tests.

Reply:
Sounds good.